src/app/components/world-map/world-map.component.ts

```diff
@@ -0,0 +1,44 @@
+import { AfterViewInit, Component, Inject } from '@angular/core';
+import * as L from 'leaflet';
```

Comment:
Prefer meaningful names instead of single letters

Reply:
You have a point, but single-letter imports are surprisingly common.
Examples on leafletjs.com all import is as L, and we see similar patterns for other modules (e.g. ramda is imported as `R`, lodash is imported as `_` etc.).
It's a weird convention that doesn't make a lot of sense in this day and age.

Instead of importing `*` here, I think it's better to just import `{ map, titleLayer }` if it works.